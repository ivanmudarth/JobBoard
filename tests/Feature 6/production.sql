-- Feature 6

-- User ivanm(id = 3) adds two reviews to employer with id=2
INSERT INTO Review VALUES (2, 3, 'this is my first review', 3, '2023-3-12 12:23:34')
INSERT INTO Review VALUES (2, 3, 'this is my second review', 4, '2023-3-13 1:20:30')

SELECT * FROM Review

-- View all reviews of employer with id=2
SELECT user_id, review_text, employer_rating, timestamp
FROM Review 
WHERE employer_id = 2
ORDER BY timestamp DESC