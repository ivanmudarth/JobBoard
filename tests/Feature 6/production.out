Execute:
> SELECT * FROM Review

+ ---------------- + ------------ + ---------------- + ------------------ + -------------- +
| employer_id      | user_id      | review_text      | review_rating      | timestamp      |
+ ---------------- + ------------ + ---------------- + ------------------ + -------------- +
| 2                | 3            | this is my first review | 3                  | 2023-03-12 12:23:34 |
| 2                | 3            | this is my second review | 4                  | 2023-03-13 01:20:30 |
+ ---------------- + ------------ + ---------------- + ------------------ + -------------- +
2 rows


Execute:
> SELECT user_id, review_text, review_rating, timestamp
FROM Review 
WHERE employer_id = 2
ORDER BY timestamp DESC

+ ------------ + ---------------- + ------------------ + -------------- +
| user_id      | review_text      | review_rating      | timestamp      |
+ ------------ + ---------------- + ------------------ + -------------- +
| 3            | this is my second review | 4                  | 2023-03-13 01:20:30 |
| 3            | this is my first review | 3                  | 2023-03-12 12:23:34 |
+ ------------ + ---------------- + ------------------ + -------------- +
2 rows